ui/admin/app/components/model/DefaultModelAliasForm.tsx

```diff
@@ -135,21 +155,21 @@ export function DefaultModelAliasForm({
 
     return (
         <Form {...form}>
-            <form onSubmit={handleSubmit} className="space-y-6">
+            <form onSubmit={handleFormSubmit} className="space-y-6">
                 {sortedDefaultAliases?.map(({ alias, model: defaultModel }) => (
                     <FormField
-                        control={form.control}
+                        control={control}
                         name={alias}
                         key={alias}
                         render={({ field: { ref: _, ...field } }) => {
-                            const usage = getModelUsageFromAlias(alias);
-                            const modelOptions = usage
-                                ? modelUsageMap.get(
-                                      usage === ModelUsage.Vision
-                                          ? ModelUsage.LLM
-                                          : usage
-                                  )
-                                : [];
+                            const usage =
+                                getModelUsageFromAlias(alias) ??
+                                ModelUsage.Unknown;
+
+                            const modelOptions = filterModelsByUsage(
```

Comment:
nit: Just a thought but wondering if a usage is set, should the options show both ones marked as `usage` and the unknowns? In case a type is set for one, but maybe a user wanted to find one of the unknowns?

Reply:
check the `renderSelectModelContent` method :)